docs/docs/guides/tracking/tracing.md

```diff
@@ -97,3 +97,55 @@ sentence = "There are many fruits that were found on the recently discovered pla
 with weave.attributes({'user_id': 'lukas', 'env': 'production'}):
     extract_fruit(sentence)
 ```
+
+## Customize display names
+
+You can customize display names to better identify calls by setting:
+
+1. The `display_name` parameter in the `@weave.op` decorator
+
+   ```python
+   @weave.op(display_name="custom_name")
+   def func():
+       ...
+   ```
+
+2. Or, the `display_name` attribute on the function object itself.
+
+   ```py
+   func.display_name = "custom_name"
+   ```
+
+3. The `display_name` can also be a function that takes in a `Call` object and returns a string. The `Call` object will be passed automatically when the function is called, so you can use it to dynamically generate names based on call inputs, attributes, etc.
+
+   ```py
+   def custom_attribute_name(call):
+       model = call.attributes["model"]
+       revision = call.attributes["revision"]
+       now = call.attributes["date"]
+
+       return f"{model}__{revision}__{now}"
```

Comment:
One common pattern I would think is just taking the fn name and adding a ts string or similar. Can we have the example show that? I wouldn't know from looking at this doc how to pull the "op name" out of the "call" object.

Reply:
I don't think this is easy to do atm, but you're right we should make it easy and show users how to do it.

Basically right now the op_name will be the fully qualified weave ref, which is probably not what the user wants.

Reply:
Really? How does the op name eventually get attached to the call then?

Reply:
`op_name` is the ref atm:
![image](https://github.com/user-attachments/assets/657ceee8-2c4c-41b0-8d3e-395f642461b7)

I'm assuming then in the UI it's parsed

Reply:
@tssweeney is it safe to parse out a friendly op name here?

Reply:
`op_name` is defined as either:
1. (most common case): A ref pointing to the source op
2. a simple string

The second case is what we loosely refer to as an "anonymous call" where we don't have a pointer to a producing Op.

Therefore "op_name" is probably not the best field name...

It is not safe to abandon that, but it is safe to parse it out. This is in fact what we do in the UI.

Reply:
Added a property `func_name` on call that will do the parsing for users.